targets/gnome

```diff
@@ -10,7 +10,7 @@ CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 
 ### Append to prepare.sh:
 install --minimal evolution-data-server gnome-control-center \
-        gnome-screensaver gnome-session gnome-session-fallback \
+        gnome-screensaver gnome-session gnome-session-flashback \
```

Comment:
@dnschneid  thats right, that just occurred to me right after I did the pull request, since the flashback seems to be a package that will be used as the default for the latest releases of ubuntu and GNOME and onward, but not necessarily the case for the previous prior to trusty. I'll give the alternative a shot. Thanks!